assets/components/directDebit/directDebitForm/directDebitForm.scss

```diff
@@ -1,79 +1,177 @@
 .component-direct-debit-form {
-  width: 500px;
-  padding: $gu-h-spacing/2;
+  clear: left;
+  margin-top: 80px;
+  margin-left: 0;
+
+  @include mq($from: mobileLandscape) {
+    margin-left: $gu-h-spacing;
+  }
+
+  .component-error-message {
+    background-color: transparent;
+    color: gu-colour(error);
+    padding: 0;
+    height: 0;
+  }
+
+  .svg-exclamation-alternate {
+    float: left;
+    margin-right: 8px;
+  }
+
+  .component-error-message__message {
+    position: relative;
+    left: 1px;
+    top: 3px;
+  }
 }
 
 .component-direct-debit-form__field-label {
   display: block;
-  font-family: $gu-sans-web;
-  font-size: 16px;
-  line-height: 24px;
+  font-family: $gu-egyptian-web;
+  font-weight: 600;
+  font-size: 14px;
+  margin-top: 8px;
+  margin-bottom: 6px;
 }
 
 .component-direct-debit-form__sort-code-field {
   font-family: $gu-sans-web;
   font-size: 14px;
+  background-color: gu-colour(garnett-neutral-3);
   color: gu-colour(garnett-neutral-1);
-  margin: 0 -2px 20px;
-  border: none;
-  border-radius: 600px;
-  height: $gu-v-spacing * 4;
-  padding: 0 $gu-h-spacing;
-  width: 70px;
+  border: 1px solid gu-colour(garnett-neutral-1);
+  border-radius: 25px;
+  height: 41px;
+  width: 57px;
+  padding: 0 15px;
+  margin-bottom: 25px;
+}
+
+.component-direct-debit-form__sort-code-separator {
+  margin: 0 10px;
+  font-size: 18px;
 }
 
 .component-direct-debit-form__text-field {
   font-family: $gu-sans-web;
   font-size: 14px;
-  color: gu-colour(garnett-neutral-1);
-  margin: 0 -2px 20px;
-  border: none;
-  border-radius: 600px;
-  height: $gu-v-spacing * 4;
-  padding: 0 $gu-h-spacing;
-  width: 100px;
+  font-weight: bold;
+  background-color: gu-colour(garnett-neutral-3);
+  color: black;
+  border: 1px solid gu-colour(garnett-neutral-1);
+  border-radius: 25px;
+  height: 40px;
+  width: 313px;
+  @include mq($from: mobileLandscape) {
+    width: 405px;
+  }
+  padding: 0 15px;
 }
 
-.component-direct-debit-form__account-holder-confirmation__text {
+.component-direct-debit-form__confirmation-text {
   font-family: $gu-sans-web;
   font-size: 14px;
   color: gu-colour(neutral-1);
+  padding-left: 33px;
+  display: block;
+  overflow: hidden;
 }
 
+.component-direct-debit-form__confirmation-css-checkbox {
+  width: 25px;
+  position: relative;
+  margin-right: 10px;
+}
+
+.component-direct-debit-form__confirmation-label {
+  cursor: pointer;
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  top: 2px;
+  left: 0;
+  background: #fff;
+  border: solid 1px #121212;
+  border-radius: 2px;
+}
+
+.component-direct-debit-form__confirmation-label:after {
+  opacity: 0;
+  content: '';
+  position: absolute;
+  width: 9px;
+  height: 5px;
+  background: transparent;
+  top: 4px;
+  left: 4px;
+  border: 3px solid #333;
+  border-top: none;
+  border-right: none;
+  transform: rotate(-45deg);
+}
+
+.component-direct-debit-form__confirmation-label:hover::after {
+  opacity: 0.5;
+}
```

Comment:
I'm not suggesting that you change this, but you could use `&:hover` and `&:after` within the `.component-direct-debit-form__confirmation-label` rule, so that everything for this class is in one place.

This isn't a codified convention for this codebase; @svillafe have discussed it but didn't come to a conclusion.

Reply:
Yup, that is nicer.